### DISCUSSION TITLE
Support for Qt qml

### DISCUSSION THREAD

**Discussion by FreeSunProgram** [CefView/QCefView#417] [Q&A]
have you considered rendering cef off-screen to QQuickItem to support the use of cef in QML pages?

**Comment by tishion** [comment]
No.

actually, I am considering stopping the maintenance of this project as CEF always introduces breaking changes a lot.

**Reply by EdnY1** [reply]
Don't drop it please, :( this is such a great app engine. And about qml, I would never attempt to support QML, Its full of issues
